save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/docker/DockerAgentRunner.kt

```diff
@@ -213,6 +212,7 @@ class DockerAgentRunner(
         val envFile = createTempDirectory("orchestrator").resolve(".env").apply {
             writeText("""
                 ${AgentEnvName.AGENT_ID.name}=$containerId
+                ${AgentEnvName.AGENT_NAME.name}=$containerName
```

Comment:
`containerName` is available before running the docker agent, it can be moved to `.withEnv()`